service/src/main/kotlin/fi/espoo/evaka/reports/ServiceVoucherValueReport.kt

```diff
@@ -171,6 +172,7 @@ data class ServiceVoucherValueRow(
     val serviceVoucherCoPayment: Int,
     val serviceVoucherServiceCoefficient: Int,
     val serviceVoucherHoursPerWeek: Int,
+    val placementType: PlacementType,
```

Comment:
This is actually not the correct enum to use here, it should be `fi.espoo.evaka.invoicing.domain.PlacementType` (yes, it's pretty confusing).